app/src/main/java/com/arygm/quickfix/ui/search/SearchWorkerProfileResult.kt

```diff
@@ -135,7 +144,7 @@ fun SearchWorkerProfileResult(
                             tint = colorScheme.onSurface,
                             modifier = Modifier.size(16.dp))
```

Comment:
I know it is not your code, but if you can change that to be relative it would be nice 😄 (otherwise just opening an issue for it should be fine for now)